pytest_django/live_server_helper.py

```diff
@@ -1,56 +1,59 @@
-from typing import Any, Dict
-
-
 class LiveServer:
     """The liveserver fixture
 
     This is the object that the ``live_server`` fixture returns.
     The ``live_server`` fixture handles creation and stopping.
     """
 
-    def __init__(self, addr: str) -> None:
+    def __init__(self, addr: str, start: bool = True) -> None:
+        from django.conf import settings
+        from django.contrib.staticfiles.handlers import StaticFilesHandler
         from django.db import connections
-        from django.test.testcases import LiveServerThread
+        from django.test.testcases import LiveServerThread, _StaticFilesHandler
         from django.test.utils import modify_settings
 
-        liveserver_kwargs = {}  # type: Dict[str, Any]
+        try:
+            host, port_str = addr.split(":")
+            port = int(port_str)
+        except ValueError:
+            host = addr
+            port = 0
```

Comment:
I think the entire `liveserver_kwargs` business was to avoid this line, i.e. not duplicate Django's default in case it changes.